### DISCUSSION TITLE
BetterEnd/BetterNether compatibility

### DISCUSSION THREAD

**Discussion by emeraldragon10005** [Sinytra/ForgifiedFabricAPI#92] [Ideas]
Hi, maybe a dumb topic to open but these two mods I think add a lot to Minecraft's gameplay and it's a shame they are strictly Fabric only... i tried them just now and the game seems to launch just fine but doesn't read the mods or its prerequisite, BCLib, to be in the game anywhere

I've been waiting for an opportunity to try this out and finally play with the modlist i've been wanting to with these two mods

**Comment by emeraldragon10005** [comment]
Hello, i dont know if you read or added this suggestion but the latest version of the mod works with BetterEnd/BetterNether... i never thought this was possible you guys are fantastic for doing this, many thanks

**Comment by Dokman** [comment]
in my end it doesn't detect the mods idk why

**Comment by SOSSALMON** [comment]
yeah, I'm having an issue with it aswell
